lib/routes/cags/edu/index.ts

```diff
@@ -0,0 +1,84 @@
+import ofetch from '@/utils/ofetch';
+import { Route } from '@/categorys';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const host = 'https://edu.cags.ac.cn';
+
+const titles = {
+    tzgg: '通知公告',
+    ywjx: '要闻简讯',
+    zs_bss: '博士生招生',
+    zs_sss: '硕士生招生',
+    zs_dxsxly: '大学生夏令营',
+};
+
+export const route: Route = {
+    path: '/edu/:category',
+    categories: ['university'],
+    example: '/cags/edu/tzgg',
+    parameters: {
+        category: '通知频道，可选 tzgg/ywjx/zs_bss/zs_sss/zs_dxsxly',
+    },
+    features: {
+        antiCrawler: false,
+        requireConfig: false,
+        requirePuppeteer: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '中国地质科学院研究生院',
```

Comment:
https://github.com/DIYgod/RSSHub/pull/17458#discussion_r1829433331

> No need to repeat namespace name in route name

```suggestion
    name: '研究生院',
```